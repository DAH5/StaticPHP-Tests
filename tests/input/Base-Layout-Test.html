---
page_title: Hello, world!
layout: tests/input/_includes/layout.html
---
<p>This is the page content. It should be surrounded by a normal layout.</p>