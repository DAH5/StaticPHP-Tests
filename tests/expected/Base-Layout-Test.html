<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        <title>Hello, world! :: Base Layout Test</title>
    </head>

    <body>
        <h1>Base Layout Test</h1>
        <p>Hello, world!</p>

        <p>This is the page content. It should be surrounded by a normal layout.</p>

        <p>Copyright &copy; StaticPHP Developer.</p>
    </body>
</html>